packages/design-tokens/design-token-previews/utils/previews.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import includes from "lodash/includes";
+
+import { TokenEntry } from "../types/TokenEntry";
+
+import { FontSizePreview } from "../FontSizePreview";
+import { FontWeightPreview } from "../FontWeightPreview";
+import { IconSizePreview } from "../IconSizePreview";
+
+export const createFontWeightPreview = ([
+  tokenSuffix,
+  token,
+]: TokenEntry): JSX.Element => {
+  return <FontWeightPreview fontWeight={token.value} key={tokenSuffix} />;
```

Comment:
Why do we need a component for this one and not use it directly as you did with the others?